doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -14,9 +14,17 @@ GNU ARM Embedded
 
    .. warning::
 
-      Like the Zephyr repository, do not install the toolchain in a directory
-      with spaces anywhere in the path. On Windows, we'll assume you install
-      into the directory :file:`C:\\gnu_arm_embedded`.
+      Do not install the toolchain into a path with spaces. On
+      Windows, we'll assume you install into the directory
+      :file:`C:\\gnu_arm_embedded`.
+
+   .. warning::
+
+      The Windows release of 8-2018-q4-major has a critical bug and
```

Comment:
Just to be clear that we're talking about the GNU ARM Embedded Toolchain for Windows version (and not the version of Windows), how about:
```
The GNU ARM Embedded Toolchain for Windows, version **8-2018-q4-major**
has a `critical bug <https://github.com/zephyrproject-rtos/zephyr/issues/12257>`_
and should not be used. Toolchain version **7-2018-q2-update** is known to work.
```

Reply:
Done.

And thank you, much better.